docs/user/hbase/visibilities.rst

```diff
@@ -13,13 +13,13 @@ Setup and Configuration
 
 To configure HBase for visibility filtering follow the setup in the HBase Book under the
 `Visibility Labels <http://hbase.apache.org/book.html#hbase.visibility.labels>`__ section of
-the HBase book which includes enabling the HFile v3 format and visibility coprocessors in your hbase-site.xml:
+the HBase book which includes enabling the visibility coprocessors in your hbase-site.xml:
 
 .. code-block:: xml
 
     <property>
-      <name>hfile.format.version</name>
-      <value>3</value>
```

Comment:
Does the hfile format version requirement drop out?  Is it still necessary for HBase 1.4?

Reply:
i'm pretty sure 3 has been the default for a long time, it was first introduce in hbase 0.98. I verified it's the default at least back to 1.4.